packages/block-library/src/latest-comments/index.php

```diff
@@ -5,7 +5,7 @@
  * @package WordPress
  */
 
-if ( ! function_exists( 'gutenberg_draft_or_post_title' ) ) {
+if ( ! function_exists( 'wp_latest_comments_draft_or_post_title' ) ) {
```

Comment:
We should remove this check. We don't do pluggable functions in core anymore. Plus, it's the only `functions_exists()` check in this file.

Reply:
👍

Reply:
Will this create issues when we run Gutenberg (the plugin) in Core? Loading the same function twice? We'll have to check.